_posts/en/newsletters/2025-03-07-newsletter.md

```diff
@@ -0,0 +1,229 @@
+
+  His post notes that "it’s only a guide to the cooperative path, where
+  you make a change that makes everyone’s life better, and more or less
+  everyone ends up agreeing that the change makes everyone’s lives
+  better."  He also warns that "it's also only a fairly high-level
+  guide."
+
+- **Update on BIP360 pay-to-quantum-resistant-hash (P2QRH):** developer
+  Hunter Beast [posted][beast p2qrh] an update on his research into
+  [quantum resistance][topic quantum resistance] for [BIP360][] to the
+  Bitcoin-Dev mailing list.  He's made changes to the list of
+  quantum-secure algorithms he's proposing to use, is looking for
+  someone to champion development of a pay-to-taproot-hash (P2TRH)
+  scheme (see [Newsletter #141][news141 p2trh]), and is considering
+  targeting the same security level as currently provided by Bitcoin
+  (NIST II) rather than a higher level (NIST V) that requires more block
+  space and CPU verification time.  His post received multiple replies.
+
+- **Private block template marketplace to prevent centralizing MEV:** Matt
+  Corallo and developer 7d5x9 [posted][c7 mev] to Delving Bitcoin about
+  allowing parties to bid in a public markets for selected space within
+  miner block templates.  For example, "I’ll pay X [BTC] to include
+  transaction Y as long as it comes before any other transactions which
+  interact with the smart contract identified by Z".  This is something
+  that transaction creators on Bitcoin already want for various
+  protocols, such as certain [colored coin protocols][topic client-side
+  validation], and it's likely to become even more desirable in the
+  future as new protocols are developed (including proposals requiring
+  consensus changes such as certain [covenants][topic covenants]).
+
+  If the service of preferential transaction ordering within block
+  templates is not provided by a trust-reduced public market, it's
+  probable that it will instead be provided by large miners, who will
+  compete with users of the various protocols.  This will require the
+  miners obtain large amounts of capital and technical sophistication,
+  likely leading to them earning significantly higher profits than smaller
+  miners without those capabilities.  This will centralize mining and
+  allow the large miners to more easily censor Bitcoin transactions.
+
+  The developers propose providing trust reduction by allowing miners to
+  work on blinded block templates whose complete transactions aren't
+  revealed to the miner until they've produced sufficient proof of work
+  to publish the block.  The developers propose two mechanisms for
+  achieving this without requiring any consensus changes:
+
+  - **Trusted block templates:** a miner connects to a marketplace, selects
+    the bids it wants to include in a block, and asks the marketplace to
+    construct a block template.  The marketplace responds with a block
+    header, coinbase transaction, and partial merkle branch that allow
```

Comment:
```suggestion
    header, coinbase transaction, and partial merkle branch, allowing
```

Reply:
Keeping as-is.  I think both constructions are correct and the existing construction avoids the use of a comma shortly after a comma-delimited list, making the sentence easier to skim.